tests/fixtures/s3_success_response.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PostResponse>
+    <Location>http://pydocx.s3.amazonaws.com/uploads/pydocx/image1.png</Location>
+    <Bucket>pydocx</Bucket>
+    <Key>uploads/pydocx/image1.png</Key>
+    <ETag>"d41d8cd98f00b204e9800998ecf8427e"</ETag>
+</PostResponse>
```

Comment:
#nitpick - newline at end of file missing

Reply:
fixed - 3bd3821a3470c153127e87f5ee6073a7f3b6fda3